axelor-account/src/main/java/com/axelor/apps/account/service/AccountService.java

```diff
@@ -68,4 +71,30 @@ public BigDecimal computeBalance(Account account, int balanceType) {
       return BigDecimal.ZERO;
     }
   }
+  
+  public List<BigDecimal> computDebitCredit(Account account) {
+	  
+    Query balanceQuery =
+    JPA.em()
+        .createQuery(
+            "select sum(self.debit),sum(self.credit) from MoveLine self where self.account = :account "
+                + "and self.move.ignoreInAccountingOk IN ('false', null) and self.move.statusSelect IN (2, 3)");
+
+    balanceQuery.setParameter("account", account);
+    
+    Object[] debitCredit = (Object[]) balanceQuery.getResultList().get(0); 
+    
+    if(debitCredit[0] == null && debitCredit[1] == null) {
+    	return null;
+    } else {
+    	
+    	List<BigDecimal> debitCreditLst = new ArrayList<>();
+    	
+    	debitCreditLst.add(new BigDecimal(debitCredit[0].toString()));
+    	debitCreditLst.add(new BigDecimal(debitCredit[1].toString()));
+    	
+    	return debitCreditLst;
```

Comment:
return map with 'credit' and 'debit' as its keys instead of list, it will be more readable